scala/README.md

```diff
@@ -284,13 +325,28 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     val answer = ys exists (...)
     ```
 
-0. Avoid wildcard-importing entire packages: `import package._`
+0. Avoid implicit conversions, use implicit value classes (extension methods) instead. In other words, do not import `scala.language.implicitConversions`:
```

Comment:
link value classes to http://docs.scala-lang.org/overviews/core/value-classes.html ?